components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/token/DefaultApiKeyGenerator.java

```diff
@@ -221,4 +232,17 @@ private static String encode(byte[] stringToBeEncoded) throws APIManagementExcep
             throw new APIManagementException("Error while encoding the Api Key ",e);
         }
     }
+
+    private static boolean isLightweightAPIKeyGenerationEnabled() {
+        try {
```

Comment:
Duplicated the same method within the same package. Shall we make this reusable?

Reply:
I have implemented a new Util class to hold APIKey Utils and included a method to read the config.